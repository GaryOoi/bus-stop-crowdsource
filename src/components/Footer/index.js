import React from "react";

function Footer() {
  const year = new Date().getFullYear();
  return (
    <div className="text-center mt-5 bg-dark">
      <div className="border-top p-4">
        <div className="text-light">
          Copyright &copy;
          {year}
        </div>
      </div>
    </div>
  );
}

export default Footer;
