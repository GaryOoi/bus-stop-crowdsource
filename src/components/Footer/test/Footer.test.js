import React from "react";
import { render } from "@testing-library/react";

import Footer from "../index";

test("renders learn react link", () => {
  const { container } = render(<Footer />);

  expect(container.firstChild).toMatchSnapshot();
});
