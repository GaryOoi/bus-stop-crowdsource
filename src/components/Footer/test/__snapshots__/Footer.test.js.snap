// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="text-center mt-5 bg-dark"
>
  <div
    class="border-top p-4"
  >
    <div
      class="text-light"
    >
      Copyright Â©
      2020
    </div>
  </div>
</div>
`;
